crates/pathfinder/src/rpc/v02/method/block_hash_and_number.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::Context;
+use stark_hash::StarkHash;
+
+use crate::{rpc::v02::RpcContext, storage::StarknetBlocksTable};
+
+type BlockNumber = u64;
+
+#[derive(serde::Serialize)]
+pub struct BlockNumberResult {
```

Comment:
Minor nit, and only because `Result` is often used for fallible return values i.e. `Result<T, E>`.
```suggestion
pub struct BlockNumberOuput {
```

Reply:
Done, will avoid *Result for DTOs.